app/scripts/controllers/landing.js

```diff
@@ -41,6 +41,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
+        analytics.digitalData.searchTeam = val;
```

Comment:
@adammeyer Did this get addressed? Is it correct as is or is it a spelling error?